src/docs/data-sources.dox

```diff
@@ -67,9 +64,9 @@ Advanced applications may also open the following low-level cursor types:
 WiredTiger provides access to data from a variety of sources.  At the
 lowest level, data may be stored in a file using a tree structure.  A
 relational schema supporting tables, indices and column groups is
-layered on top of file.  Additional sources include LSM trees and
-statistics, and applications can further extend the supported types by
-implementing the ::WT_DATA_SOURCE interface.
+layered on top of file. Additional sources include statistics.
```

Comment:
Does it make sense to day "Additional sources include statistics, log files and backups"?

Reply:
Sure, sounds good. See [8f07ddc](https://github.com/wiredtiger/wiredtiger/pull/11299/commits/8f07ddc25363fc67b6183a9f8a60be26a3430309)